docs/Makefile

```diff
@@ -0,0 +1,179 @@
+# Minimal makefile for Sphinx documentation
+#
+# Add your customisation to `Makefile` instead.
+
+# You can set these variables from the command line, and also
+# from the environment for the first two.
+SPHINXDIR       = .sphinx
+SPHINXOPTS      ?= -c . -d $(SPHINXDIR)/.doctrees -j auto
+SPHINXBUILD     ?= $(VENVDIR)/bin/sphinx-build
+SOURCEDIR       = .
+BUILDDIR        = _build
+VENVDIR         = $(SPHINXDIR)/venv
+PA11Y           = $(SPHINXDIR)/node_modules/pa11y/bin/pa11y.js --config $(SPHINXDIR)/pa11y.json
+VENV         	   = $(VENVDIR)/bin/activate
+TARGET          = *
+ALLFILES        =  *.rst **/*.rst
+METRICSDIR      = $(SOURCEDIR)/.sphinx/metrics
+REQPDFPACKS     = latexmk fonts-freefont-otf texlive-latex-recommended texlive-latex-extra texlive-fonts-recommended texlive-font-utils texlive-lang-cjk texlive-xetex plantuml xindy tex-gyre dvipng
+CONFIRM_SUDO    ?= N
+VALE_CONFIG     = $(SPHINXDIR)/vale.ini
+
+# Put it first so that "make" without argument is like "make help".
+help:
+	@echo "\n" \
+        "------------------------------------------------------------- \n" \
+        "* watch, build and serve the documentation:  make run \n" \
+        "* only build:                                make html \n" \
+        "* only serve:                                make serve \n" \
+        "* clean built doc files:                     make clean-doc \n" \
+        "* clean full environment:                    make clean \n" \
+        "* check links:                               make linkcheck \n" \
+        "* check spelling:                            make spelling \n" \
+        "* check spelling (without building again):   make spellcheck \n" \
+        "* check inclusive language:                  make woke \n" \
+        "* check accessibility:                       make pa11y \n" \
+        "* check style guide compliance:              make vale \n" \
+        "* check style guide compliance on target:    make vale TARGET=* \n" \
+        "* check metrics for documentation:           make allmetrics \n" \
+        "* other possible targets:                    make <TAB twice> \n" \
+        "------------------------------------------------------------- \n"
+
+.PHONY: full-help woke-install spellcheck-install pa11y-install install run html \
+        epub serve clean clean-doc spelling spellcheck linkcheck woke \
+        allmetrics pa11y pdf-prep-force pdf-prep pdf Makefile.sp vale-install vale
+
+full-help: $(VENVDIR)
+	@. $(VENV); $(SPHINXBUILD) -M help "$(SOURCEDIR)" "$(BUILDDIR)" $(SPHINXOPTS) $(O)
+	@echo "\n\033[1;31mNOTE: This help texts shows unsupported targets!\033[0m"
+	@echo "Run 'make help' to see supported targets."
+
+# If requirements are updated, venv should be rebuilt and timestamped.
+$(VENVDIR):
+	python3 -c "import venv" || \
```

Comment:
Would be fun to fix this Makefile to use `uv` (and see how much it speeds up!). And then see if upstream wants to take the changes. :-) I've opened https://github.com/canonical/jubilant/issues/43 to rack.